src/main/webapp/pages/projects/analyses/pages/analyses_table.html

```diff
@@ -1,15 +1,34 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org"
-      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      data-layout-decorate="~{projects/analyses/_base}">
-
-<body>
-
-<th:block layout:fragment="analyses-content">
-    <th:block th:replace="analyses/analyses-table :: table"/>
-</th:block>
-<th:block layout:fragment="scripts">
-    <th:block th:replace="analyses/analyses-table :: scripts"/>
-</th:block>
-</body>
+<html
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  data-layout-decorate="~{projects/analyses/_base}"
+>
+  <head>
+    <link rel="stylesheet" th:href="@{/dist/css/project-analyses.bundle.css}" />
+    <script th:inline="javascript">
+      window.PAGE = {
+        admin: /*[[${isAdmin or isOwner}]]*/ true,
```

Comment:
Shouldn't the default be set to false rather than true here as well?

Reply:
Zen here too